src/MarketingPageAcceptanceTests.Actions/Pages/Dashboard.cs

```diff
@@ -125,6 +125,8 @@ public void NavigateToSection(string sectionTitle, bool subDashboard = false)
         /// </summary>
         public void NavigateToPreviewPage()
         {
+            wait.Until(s => s.FindElement(pages.Dashboard.PreviewPage).Displayed);
+            wait.Until(s => s.FindElement(pages.Dashboard.PreviewPage).Enabled);
```

Comment:
this could be condensed to a single wait 
`wait.Until(s => s.FindElement(pages.Dashboard.PreviewPage).Displayed && s.FindElement(pages.Dashboard.PreviewPage).Enabled);`